dev/src/util.ts

```diff
@@ -222,10 +222,10 @@ export function wrapError(err: Error, stack: string): Error {
  * Parses the value of the environment variable FIRESTORE_PREFER_REST, and returns
  * a value indicating if the environment variable enables or disables preferRest.
  *
- * This method will warn to the console if the environment variable is set to an unsupported value.
+ * This function will warn to the console if the environment variable is set to an unsupported value.
```

Comment:
nit: wrap at 80 characters

Reply:
FWIW, (gts)[https://github.com/google/gts] enforces Google's typescript style guide and does not enforce that column limit on comments and some other line types. It's unclear if that's because there's an issue or the 80-column limit does not actually apply.